s3/proto/s3.proto

```diff
@@ -273,7 +268,7 @@ message Bucket {
 	string id = 6;
 	bool deleted = 7;
 	ServerSideEncryption serverSideEncryption = 8;
-	string versioning = 9;
+	BucketVersioning versioning = 9;

```

Comment:
Same question, only status need for versioning, why add a new struct here?

Reply:
addressed above